src/Command/CodeStudio/CodeStudioCommandTrait.php

```diff
@@ -273,4 +274,16 @@ private function getGitLabProjectDefaults(): array {
     ];
   }
 
+  /**
+   * Add gitlab options to the command.
+   *
+   * @return $this
+   */
+  protected function acceptGitlabOptions(): static {
```

Comment:
This is an easy mutant to kill, just make this method private if it's not being overridden anywhere.

Reply:
Fixed!